pocs/focuser/birger.py

```diff
@@ -260,16 +266,22 @@ def move_to(self, position):
         Does not do any checking of the requested position but will warn if the lens reports hitting
         a stop.
         """
-        response = self._send_command('fa{:d}'.format(int(position)), response_length=1)
-        if response[0][:4] != 'DONE':
-            self.logger.error("{} got response '{}', expected 'DONENNNNN,N'!".format(
-                self, response[0].rstrip()))
-        else:
-            r = response[0][4:].rstrip()
-            self.logger.debug("Moved to {} encoder units".format(r[:-2]))
-            if r[-1] == '1':
-                self.logger.warning('{} reported hitting a focus stop'.format(self))
-            return int(r[:-2])
+        self._is_moving = True
+        try:
+            response = self._send_command('fa{:d}'.format(int(position)), response_length=1)
+            if response[0][:4] != 'DONE':
+                self.logger.error("{} got response '{}', expected 'DONENNNNN,N'!".format(
+                    self, response[0].rstrip()))
+            else:
+                r = response[0][4:].rstrip()
+                self.logger.debug("Moved to {} encoder units".format(r[:-2]))
```

Comment:
```suggestion
                self.logger.debug("Moved to {} encoder position".format(r[:-2]))
```

Reply:
Opted for "Moved to encoder position {}"